examples/pipelines/simple_pipeline/components/image_filtering/src/fondant_component.yaml

```diff
@@ -0,0 +1,27 @@
+name: Image filtering
```

Comment:
I think the image width and height can be int16 (max value of 32767). Saves us some memory for the parquet file (not relevant now but might be more relevant with large scale datasets)

Reply:
Thanks, have added a todo for that, currently only the appropriate column names are checked.